File: 1069.py
n = int(input())

for i in range(n):
    balance = 0
    contador = 0
    expressao = input()
    tamExp = len(expressao)

    for j in range(tamExp):
        if expressao[j] == '<':
            balance += 1
        elif balance > 0 and expressao[j] == '>':
            balance -= 1
            contador += 1

    print(contador)

File: 1069.cpp
#include <iostream>
using namespace std;

int main() {
    int i, j, n;
    string expressao;

    cin >> n;
    for(i = 0; i < n; i++) {
        int balance = 0;
        int contador = 0;

        cin >> expressao;
        int tamExp = expressao.length();
        for (j = 0; j < tamExp; j++) {
            if (expressao[j] == '<') {
                balance++;
            } else if (balance > 0 && expressao[j] == '>') {
                balance--;
                contador++;
            }
        }
        cout << contador << "\n";
    }

    return 0;
}
